examples/gh-pages/scripts/components/events/ClosureListeners.js

```diff
@@ -49,6 +49,10 @@ class ClosureListeners extends React.Component {
     this.setState(this.state);
   }
 
+  _handle_infowindow_click () {
```

Comment:
@MetalMatze this is just a test to see if it's working; all changes in this file can be removed; I know the code is rough, but I wrote this at the end of my work day today and was tired; I was hoping to get advice on improving this code, as it doesn't feel very robust; it does the job though - so I don't need to switch libs in my project; when it's a bit more refined, I'm sure it'll get in the main repo